README.md

```diff
@@ -152,9 +152,7 @@ switch!('myaccount')
 ```
 
 ## Analytics Feature
-Hyku currently only supports the configuration of one Google Analytics account for the basic functionality of this feature. Hyku currently only support Google Analytics with the Universal Analytics property for this feature.
-
-Note: Google has announced they will stop processing data using the Universal Analytics property on July 1, 2023  or July 1, 2024 for Analytics 360 properties.
+Hyku currently only supports the configuration of one Google Analytics account for the basic functionality of this feature. Hyku currently only support Google Analytics with the Analytics 4 property for this feature.
```

Comment:
I'm not following this.

Reply:
i removed the Note since i thought we're passed the UA date, not sure if anyone was using Analytics 360 but i'll bring it back